legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -496,6 +496,7 @@ Payment Activities are created when a Payment Request has been **created**, **pa
 | 400    | {% break _ CHECKSUM_FAILED %}              | Luhn checksum digit doesn't pass.                                                                         |
 | 403    | {% break _ PATRON_CODE_INVALID %}          | Patron Code doesn't exist or has expired.                                                                 |
 | 403    | {% break _ NO_AVAILABLE_PAYMENT_OPTIONS %} | No payment options match the requested payment parameters.                                                |
+| 403    | {% break _ TOKEN_COLLECTION_INVALID %}     | Either the `collectionId` is not configured with the provided token collection or the token collection does not exist.                                                                      |
```

Comment:
What does "the `collectionId` is not configured with the provided token collection" mean?

Now that I think about it, the only reason we throw this error is because the corresponding token collection does not exist. Therefore, I think the error should be `TOKEN_COLLECTION_NOT_FOUND`

Reply:
I like the new error name `TOKEN_COLLECTION_NOT_FOUND`

The reason for "the `collectionId` is not configured with the provided token collection" is that there might be a scenario where the collectionId is valid but hasn't yet been set up/configured to a token collection.

 I agreed that we could implement this later and we will focus on `TOKEN_COLLECTION_NOT_FOUND` with the description of 'The token collection does not exist.' 👍